openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -93,7 +96,7 @@ def setUp(self):
         self.ACCESS_TOKEN = 'access_token'  # pylint: disable=invalid-name
         self.REFRESH_TOKEN = 'refresh_token'  # pylint: disable=invalid-name
 
-        self.oauth2_client = oauth2_provider.oauth2.models.Client.objects.create(
+        self.dop_oauth2_client = oauth2_provider.oauth2.models.Client.objects.create(
```

Comment:
Why not call the `dop_adapter.create_public_client` helper to create the client?